src/Data/Hours/DateData.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lits\LibCal\Data\Hours;
+
+use Lits\LibCal\Data;
+
+class DateData extends Data
+{
+    public ?string $status = null;
```

Comment:
I'd probably have $status be non-nullable as I'm guessing it is always provided.